onnxruntime/python/tools/transformers/onnx_model_phi.py

```diff
@@ -771,6 +782,7 @@ def fuse(
             subgraph_nodes.extend(self.gemm(["ln_out", attn_q_weight, attn_q_bias], ["query"], "Q_"))
             subgraph_nodes.extend(self.gemm(["ln_out", attn_k_weight, attn_k_bias], ["key"], "K_"))
             subgraph_nodes.extend(self.gemm(["ln_out", attn_v_weight, attn_v_bias], ["value"], "V_"))
+            # vllm engine requires full position ids as the input
```

Comment:
Added comments for https://github.com/microsoft/onnxruntime/pull/19429#discussion_r1482252173